app/views/posts/preview.haml

```diff
@@ -1,3 +1,7 @@
+- if @fonts.present?
```

Comment:
Plausibly this should go in the layout file somewhere so we're not throwing style tags in the body of the document